sql/2022/sustainability/green_web_hosting.sql

```diff
@@ -0,0 +1,43 @@
+#standardSQL
+# What percentage of URLs are hosted on a known green web hosting provider?
+
+WITH green AS (
+  SELECT
+    NET.HOST(url) AS host,
+    TRUE AS is_green
+  FROM
+    `httparchive.almanac.green_web_foundation`
+  WHERE
+    date = '2022-06-01'
+),
+
+pages AS (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    NET.HOST(url) AS host,
+    rank
+  FROM
+    `httparchive.summary_pages.2022_06_01_*`
+)
+
+SELECT
+  client,
+  rank_grouping,
+  COUNTIF(is_green) AS total_green,
+  COUNT(0) AS total_sites,
+  COUNTIF(is_green) / COUNT(0) AS pct_green
+FROM
+  pages
+LEFT JOIN
+  green
+USING
+  (host),
+UNNEST([1000, 10000, 100000, 1000000, 10000000]) AS rank_grouping
```

Comment:
I think this is the last linting error!

```suggestion
  UNNEST([1000, 10000, 100000, 1000000, 10000000]) AS rank_grouping
```